raiden/billing/invoices/decoder/lumino_decoder.py

```diff
@@ -0,0 +1,232 @@
+from raiden.billing.invoices.util.bech32 import bech32_decode, CHARSET, bech32_encode
+from raiden.billing.invoices.util.encoding_util import u5_to_bitarray, base58_prefix_map, bitarray_to_u5
+from raiden.billing.invoices.lumino_invoice import LuminoInvoice
+
+import bitstring
+import re
+from decimal import Decimal
+import base58
+import hashlib
+import secp256k1
+from binascii import hexlify
+
+
+def decode_lumino_invoice(coded_invoice, verbose=True):
+    hrp, data = bech32_decode(coded_invoice)
+    if not hrp:
+        raise ValueError("Bad bech32 checksum")
+
+    # BOLT #11:
+    #
+    # A reader MUST fail if it does not understand the `prefix`.
+    if not hrp.startswith('lm'):
+        raise ValueError("Does not start with ln")
```

Comment:
The error should be "Does not start with lm", and we should move the errors texts to a constant file

Reply:
ok ready